packages/api/tests/endpoints/collections/active-collections.js

```diff
@@ -23,7 +23,6 @@ const {
 const assertions = require('../../../lib/assertions');
 
 process.env.AccessTokensTable = randomId('accessTokensTable');
-process.env.CollectionsTable = randomId('collectionsTable');
```

Comment:
Were these just not removed in the endpoint tickets?

Reply:
There was no ticket to remove DynamoDB in particular for list() and activeList() endpoints, since these are only dealing with ES and PG queries, I guess. So this "remove API Collections" ticket looks to be the umbrella to cover all else, and that was also how the above got found.

Reply:
Got it, thanks for clarifying!